build/includes/terraform.mk

```diff
@@ -15,11 +15,11 @@
 ### Deploy cluster with Terraform
 terraform-init:
 	docker run --rm -it $(common_mounts) $(DOCKER_RUN_ARGS) $(build_tag) bash -c '\
-	cd $(mount_path)/build && terraform init && gcloud auth application-default login'
+	cd $(mount_path)/install/terraform && terraform init && gcloud auth application-default login'
 
 terraform-clean:
-	rm -r ./.terraform
-	rm ./terraform.tfstate*
+	rm -r ../install/terraform/.terraform
```

Comment:
Instead of using relative paths is there a variable that is the root of the repository that we could use?

Reply:
How about `$GOPATH/src/agones.dev/agones`?

Reply:
Since we support go modules we can't assume that everyone has their agones repo in that location (I don't).

Reply:
$(agones_path) to the rescue! :smile: